.github/workflows/remove-preview-deployment.yml

```
@@ -0,0 +1,36 @@
+name: Remove Preview Deployment
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - develop
+
+jobs:
+  remove_preview_deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)